source/designers-guide/google-pagespeed-best-practise/index.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: default
+shopware_version: 5.3.0
+title: Google PageSpeed Best Practise
+github_link: designers-guide/google-pagespeed-best-practise/index.md
+indexed: true
+group: Frontend Guides
+subgroup: General Resources
+menu_title: Google PageSpeed
+menu_order: 55
+---
+
+<div class="toc-list"></div>
+
+## Introduction
+
+Google PageSpeed is a great tool to get the best performance for your website. It analysis an url and gives some suggestions on how to speed up your website.
+
+This guide will cover some key metrics which have a great impact on the score and how to optimize them.
+
+## Images
+
+The size and implementation of images have a great impact on the Google PageSpeed score. If they are too big in file size or to big in dimensions for the viewport, you'll get an error like `Compressing and resizing xx could save xxKiB (xx% reduction)`.
+
+### Compression
+
+Like the error already says, your images should be compressed either using external services or with the built-in image compression and optimization in Shopware. 
+
+If your server has tools available like `optipng` or `jpegtran`, they already optimize your thumbnails using these tools. The compression of the tools is disabled by default as it can be managed in an album in the media manager.
+
+**Recommendation for the best score**
+
+* Install `optipng` on your server
+* Install `jpegtran` on your server
+* *Optional:* Run `sw:media:optimize` to optimize all existing images on the server
+
+### Thumbnails
+
+Talking about thumbnails, you may want more thumbnail sizes to give the browser more options to decide from. 
+
+**Example**
+
+An image is shown with a width of `201px` and your thumbnail sizes are `200x200` and `600x600`. The browser will then fetch the bigger `600x600` image, because the `200x200` would be too small, even if it's just 1 pixel. Google PageSpeed then complains about the requested image is too big for the given viewport.
+
+For this case, you should add more thumbnail sizes to provide the browser a more granular set of thumbnails to decide from.
+
+Keep in mind, that the more thumbnail sizes you define, the longer it will take to upload a picture to the media manager. In addition, an uploaded image requires more disk space since more thumbnail files will be generated.
+
+**Recommendation for the best score**
+
+* Generate thumbnails every 100px - 150px. `100x100`, `200x200`, `300x300`, `400x400`, ... 
+* All available thumbnail sizes are automatically provided in the view, so there is no need to modify template files. 
+
+### Responsive images
+
+Now that there are more thumbnails to choose from, the browser needs to know, how big the image that will be displayed is.
```

Comment:
I think, that this is covered in the Responsive Images document we refer to here. Otherwise I would write, that with custom themes, they should implement all available thumbnail sizes for images.